### PR TITLE
New events: 2020 Government UX Summit

### PR DESCRIPTION
**Pull request by ayerkes**
This PR implements the following **changes:**

* Creating the event page for the 3 day UX summit

**URL / Link to page**
 
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/main-2/event/2020/07/28/2020-government-ux-summit/

### CODE REVIEW

File: content/events/2020/07/2020-07-28-2020-government-ux-summit.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+# 1 -- visible
+weight: 0
+
+# Make it better ♥
+
+---
+
+Join our three-day virtual summit, **July 28 to July 30**, and gain insights on UX Methods, Service Design, and Managing UX. It’s a great opportunity to learn about UX from our federal, state, and local government UX colleagues. 
+
+{{< accordion kicker="Day 1 of the UX Summit" title="UX Methods - July 28" icon="fas fa-copy" >}}
```
Comment (saracope) [inline content/events/2020/07/2020-07-28-2020-government-ux-summit.md:60]: I think it's ok to have the day number and not "of the UX Summit" every time

File: content/events/2020/07/2020-07-28-2020-government-ux-summit.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+**Second Session 1:30 – 2:30 PM ET**
+
+ - **Ohana for Digital Service Design (or Baking Accessibility into your Digital Product Lifecycle)** - *Jennifer Strickland and Kendra Skeene, Ad Hoc (VA Modernization)*
+Designing and developing digital products that are accessible to all doesn’t just make good business sense — it’s also the law. Yet it’s often an overlooked part of the product development lifecycle. The best and most cost-effective way to ensure your product is accessible is to start with accessibility. In this session, participants will learn how to incorporate accessibility and inclusive design at each stage of the product lifecycle. From product requirements through user research, content strategy and copywriting, design, front-end development, and QA, consider accessibility using these effective guidelines. {{< /accordion >}}
+
+{{< accordion kicker="Day 2 of the UX Summit" title="Service Design - July 29" icon="fas fa-copy" >}}
+
+ **First Session –  11:30 AM – 12:30 PM ET**
+
+ - **Service Design Versus Product Design – What is it and when do we use it?** - *Caroline Kyungae Smith and Katherine Nammacher, US Digital Service*
```
Comment (saracope) [inline content/events/2020/07/2020-07-28-2020-government-ux-summit.md:85]: The bullets look weird when there's only one item in the list. I'm not sure they're necessary and would probably remove.

File: content/events/2020/07/2020-07-28-2020-government-ux-summit.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+This presentation proposes a useful method to track standards and UX requirements in an agile software development process. This presentation is also a case study of medication information standards implementation at the Veterans Health Administration.
+
+**Second Session 1:30 – 2:30 PM ET**
+
+ - **Ohana for Digital Service Design (or Baking Accessibility into your Digital Product Lifecycle)** - *Jennifer Strickland and Kendra Skeene, Ad Hoc (VA Modernization)*
+Designing and developing digital products that are accessible to all doesn’t just make good business sense — it’s also the law. Yet it’s often an overlooked part of the product development lifecycle. The best and most cost-effective way to ensure your product is accessible is to start with accessibility. In this session, participants will learn how to incorporate accessibility and inclusive design at each stage of the product lifecycle. From product requirements through user research, content strategy and copywriting, design, front-end development, and QA, consider accessibility using these effective guidelines. {{< /accordion >}}
```
Comment (saracope) [inline content/events/2020/07/2020-07-28-2020-government-ux-summit.md:79]: This one doesn't look like it's formatting correctly and maybe needs and extra space.

**Comment by ayerkes** [comment]
@saracope I think this is good to go for the most part. It is a little confusing having the single date at the top of the event page though - could we do the whole date range at the top? I don't want to "break" the page.

**Pull request merged**